### PR TITLE
[bitnami/*] Update Kubectl container version

### PR DESCRIPTION
**Pull request by FraPazGal**
**Description of the change**

Updates kubectl container's version used in several of bitnami/charts to version `1.19` given the upcoming kubectl deprecation of branches `1.16-18` per [it's deprecation notice](https://github.com/bitnami/bitnami-docker-kubectl#branch-deprecation-notice).

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/kafka/values.yaml [hunk 1]
```diff
@@ -649,7 +649,7 @@ externalAccess:
     image:
       registry: docker.io
       repository: bitnami/kubectl
-      tag: 1.17.17-debian-10-r90
+      tag: 1.19-11-debian-10-r0
```
Comment (carrodher) [inline bitnami/kafka/values.yaml:652]:
```suggestion
      tag: 1.19.11-debian-10-r0
```

File: bitnami/mongodb/values.yaml [hunk 2]
```diff
@@ -575,7 +575,7 @@ externalAccess:
     image:
       registry: docker.io
       repository: bitnami/kubectl
-      tag: 1.18.18-debian-10-r18
+      tag: 1.19-11-debian-10-r0
```
Comment (carrodher) [inline bitnami/mongodb/values.yaml:578]:
```suggestion
      tag: 1.19.11-debian-10-r0
```

File: bitnami/spring-cloud-dataflow/values.yaml [hunk 3]
```diff
@@ -757,7 +757,7 @@ waitForBackends:
   image:
     registry: docker.io
     repository: bitnami/kubectl
-    tag: 1.18.18-debian-10-r14
+    tag: 1.19-11-debian-10-r0
```
Comment (carrodher) [inline bitnami/spring-cloud-dataflow/values.yaml:760]:
```suggestion
    tag: 1.19.11-debian-10-r0
```

**Review by carrodher — APPROVED** [review]
LGTM, but before merging the PR, please fix the typo in the tags and ensure the required internal changes are implemented in
```
data-assets/common/charts-definitions/charts/kafka.jsonnet
data-assets/common/charts-definitions/charts/mongodb.jsonnet
data-assets/common/charts-definitions/charts/spring-cloud-dataflow.jsonnet
```

I just checked the image is available:
```console
$ docker pull bitnami/kubectl:1.19.11-debian-10-r0
1.19.11-debian-10-r0: Pulling from bitnami/kubectl
4fb7b694fe70: Pull complete
16511638bcb5: Pull complete
4345672d38ac: Pull complete
1db752a52c2c: Pull complete
69b4694fff4a: Pull complete
Digest: sha256:a58c5bc8981f97f0a2dc235d31d04978d196aa9e6115df1fd06fefba078b0c7c
Status: Downloaded newer image for bitnami/kubectl:1.19.11-debian-10-r0
docker.io/bitnami/kubectl:1.19.11-debian-10-r0
```

**Comment by FraPazGal** [comment]
Fixed the typo and updated internal files, thanks!

**Pull request merged**